src/lang/std/sketchcombos.ts

```diff
@@ -369,192 +373,190 @@ const setAbsDistanceCreateNode =
+    tag,
+    valueUsedInTransform,
+  })
+}
+
+const setAngledIntersectForAngledLines: CreateStdLibSketchCallExpr = ({
+  referenceSegName,
+  tag,
+  forceValueUsedInTransform,
+  inputs,
+  rawArgs: args,
+}) => {
+  const valueUsedInTransform = roundOff(Number(args[1].expr.value) || 0, 2)
+  return intersectCallWrapper({
+    fnName: 'angledLineThatIntersects',
+    angleVal: inputs[0].expr,
+    offsetVal: forceValueUsedInTransform || createLiteral(valueUsedInTransform),
+    intersectTag: createIdentifier(referenceSegName),
+    tag,
+    valueUsedInTransform,
+  })
+}
 
 const setAngleBetweenCreateNode =
-  (tranformToType: 'none' | 'xAbs' | 'yAbs'): TransformInfo['createNode'] =>
-  ({ referenceSegName, tag, forceValueUsedInTransform, varValA, varValB }) => {
-    return (args, _, referencedSegment) => {
-      const refAngle = referencedSegment
-        ? getAngle(referencedSegment?.from, referencedSegment?.to)
-        : 0
-      let valueUsedInTransform = roundOff(
-        normaliseAngle(
-          (args[0].type === 'Literal' ? Number(args[0].value) : 0) - refAngle
-        )
-      )
-      let firstHalfValue = createSegAngle(referenceSegName) as BinaryPart
-      if (Math.abs(valueUsedInTransform) > 90) {
-        firstHalfValue = createBinaryExpression([
-          firstHalfValue,
-          '+',
-          createIdentifier('HALF_TURN'),
-        ])
-        valueUsedInTransform = normaliseAngle(valueUsedInTransform - 180)
-      }
-      const binExp = createBinaryExpressionWithUnary([
+  (tranformToType: 'none' | 'xAbs' | 'yAbs'): CreateStdLibSketchCallExpr =>
+  ({
+    referenceSegName,
+    tag,
+    forceValueUsedInTransform,
+    inputs,
+    rawArgs: args,
+    referencedSegment,
+  }) => {
+    const refAngle = referencedSegment
+      ? getAngle(referencedSegment?.from, referencedSegment?.to)
+      : 0
+    let valueUsedInTransform = roundOff(
+      normaliseAngle((Number(args[0].expr.value) || 0) - refAngle)
+    )
+    let firstHalfValue = createSegAngle(referenceSegName) as BinaryPart
+    if (Math.abs(valueUsedInTransform) > 90) {
+      firstHalfValue = createBinaryExpression([
         firstHalfValue,
-        (forceValueUsedInTransform as BinaryPart) ||
-          createLiteral(valueUsedInTransform),
+        '+',
+        createIdentifier('HALF_TURN'),
       ])
-      return createCallWrapper(
-        tranformToType === 'none'
-          ? 'angledLine'
-          : tranformToType === 'xAbs'
-          ? 'angledLineToX'
-          : 'angledLineToY',
-        tranformToType === 'none'
-          ? [binExp, args[1]]
-          : tranformToType === 'xAbs'
-          ? [binExp, varValA]
-          : [binExp, varValB],
-        tag,
-        valueUsedInTransform
-      )
+      valueUsedInTransform = normaliseAngle(valueUsedInTransform - 180)
     }
+    const binExp = createBinaryExpressionWithUnary([
+      firstHalfValue,
+      (forceValueUsedInTransform as BinaryPart) ||
```

Comment:
A utility function to cast this to a BinaryPart might be useful instead of using `as`

Reply:
![image](https://github.com/user-attachments/assets/797567dd-9477-40d2-9c57-8843ddc65b27)
I've got one, that I used in the odd place her, but was trying to draw the line for the refactor somewhere, but Lee is bullying me to change these too.

Reply:
Done.